contrib/clojure-package/examples/infer/objectdetector/src/infer/objectdetector_example.clj

```diff
@@ -54,23 +54,16 @@
   "Print image detector predictions for the given input file"
   [predictions width height]
```

Comment:
width and height aren't used anymore. remove?

Reply:
If I add the w and h back into the print method it should be good and resolve it